cabal.project

```diff
@@ -17,5 +17,4 @@ package ouroboros-consensus
 source-repository-package
   type: git
   location: https://github.com/input-output-hk/iohk-monitoring-framework
-  tag: 09b269cd1926303902e8dea3c8703b6e588004f5
-
+  tag: e5336c722da520272717fd6b17c25c46bd72ebb9
```

Comment:
Is this the right one now? See https://github.com/input-output-hk/ouroboros-network/commit/aaf377b1256b46d2117f727934a7920ccd8ad58f

Reply:
Yes will remove mine.